packages/allure-cucumberjs/package.json

```diff
@@ -38,6 +38,6 @@
     "verror": "^1.10.0"
   },
   "dependencies": {
-    "allure-js-commons": "^2.0.0-beta.11"
+    "allure-js-commons": "file:../allure-js-commons"
```

Comment:
What happens when you publish this to npm? It will not work, if it stays unchanged.

Could you test it by running `npm publish --dry-run`?

Reply:
I'm pretty sure lerna will replace it

Reply:
Your bet. Good luck!